references/identity-facade/apiproxy/policies/EV-ClientCredentialsFromBody.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
```

Comment:
Could you please add the default license header here?

Reply:
Doh! Done. Totally missed that.